spec/features/home_spec.rb

```diff
@@ -14,16 +14,21 @@
   let!(:path_clinic) { create(:facility, facility_group: path_group, name: "Dr. Amir Singh") }
 
   before do
+    enable_flag(:new_permissions_system_aug_2020, admin)
     sign_in(admin.email_authentication)
-    visit root_path
+    # Root path has moved to MyFacilities#Overview
+    # visit root_path
+    visit organizations_path
```

Comment:
This isn't a home_spec anymore 🤔  Is it valuable? Should we just axe it?

Reply:
Lets leave it in for now. It tests some access things, general navigation, scoping for the user approvals partial etc. We should eventually migrate it to reflect the new home page (my facilities).